nautobot/dcim/api/nested_serializers.py

```diff
@@ -44,64 +45,70 @@
 #
 
 
-class NestedRegionSerializer(WritableNestedSerializer):
+class NestedRegionSerializer(WritableNestedSerializer, ComputedFieldModelSerializer):
```

Comment:
Personally I wouldn't expect compact/nested serializers to include computed fields, even as an opt-in - is there a particular use-case driving this?

Reply:
Removed computed fields from nested serializers.